app/views/student_flashcards/show.html.erb

```diff
@@ -1,21 +1,41 @@
-show page for student flashcard
-
-<%= link_to 'Back', student_student_flashcard_sets_student_flashcards_path, class: "btn btn-outline-dark m-2" %>
-<div class="container">
-  <div class="quiz-question-card">
-   <div>
-     <h1>Edit the student flashcard</h1>
-
-     <%= simple_form_for @student_flashcard do |f| %>
-      <%= f.input :question %>
-      <%= f.input :answer %>
-
-        <div>
-          <%= f.button :submit, class: 'btn btn-dark' %>
-          <%= link_to 'Cancel', student_flashcard_path(@student_flashcard), class: 'btn btn-outline-dark' %>
-        </div>
-      <% end %>
-     </div>
+<%= link_to 'Back', flashcard_set_flashcard_template_student_flashcard_path, class: "btn btn-outline-dark m-2" %>
+  <div class="container ">
+    <div class="card flashcard_template_card">
+      <div class="d-flex mb-3 justify-content-between">
+        <h1 style="flex-grow: 1;">
+          <%= @student_flashcard.question %>
+          <%= link_to edit_flashcard_set_flashcard_template_student_flashcard_path(@student_flashcard), class: "icon-link" do %>
```

Comment:
I guess now that we've changed the routes this path is not so long anymore?
and before you'd have to give the set and the template, besides the student flashcard